featurebyte/service/feature_materialize.py

```diff
@@ -542,10 +542,12 @@ async def _get_feast_feature_store(self) -> Optional[FeastFeatureStore]:
         return await self.feast_feature_store_service.get_feast_feature_store(feast_registry)
 
     async def _get_session(self, feature_table_model: OfflineStoreFeatureTableModel) -> BaseSession:
-        assert feature_table_model.feature_cluster is not None, "Missing feature cluster"
-        feature_store = await self.feature_store_service.get_document(
-            document_id=feature_table_model.feature_cluster.feature_store_id
-        )
+        if feature_table_model.feature_cluster is not None:
+            feature_store_id = feature_table_model.feature_store_id
```

Comment:
should this be `feature_table_model.feature_cluster.feature_store_id`?

Reply:
Yes, though there is a root validator that sets `feature_store_id` based on the feature cluster:

https://github.com/featurebyte/featurebyte/blob/ae1f0e8e89abf071b495db3405f79f4425d71d94/featurebyte/models/offline_store_feature_table.py#L85-L102

Will change back to `feature_table_model.feature_cluster.feature_store_id` to be safe and not break backward compatibility.